src/new-client/src/app/site/[site]/users/manage/page.tsx

```diff
@@ -0,0 +1,40 @@
+﻿import NhsPageTitle from '@components/nhs-page-title';
+import FindUserForm from './find-user-form';
+import { When } from '@components/when';
+import AssignRoles from './assign-roles';
+
+export type UserPageProps = {
+  params: {
+    site: string;
+  };
+  searchParams?: {
```

Comment:
Whilst I'm fine with this approach, I think it's worth pointing out that using `searchParams` automatically opts this page into dynamic rendering. In this case it means it'll render as a client component rather than on the server. If we were to switch to a dynamic route like `site/{siteId}/users/{userId}/manage` (or similar), we could fetch the specified user's details on the server and make use of SSR too. I think it's probably worth doing, but happy either way!

Reply:
Noted. I think changing it now would mean a bit of a redesign of the work as we would have to consider how to have the find user element of it (how would we do that in terms of url?) Given that this page is probably going to have less use than most of the system I think the simple approach is fine. Perhaps as we design more of the system we'll become more familiar and have a pattern we can apply to this, but for now I am inclined to leave as is

Reply:
No probs, I'm very happy to leave it for now :) 

But just to answer your point about how we'd do the find user element in urls, I think I'd have suggested something like:
`{origin}/site/{siteId}/users` -> the table of all users
`{origin}/site/{siteId}/users/manage` -> the UI to search for a user to manage
`{origin}/site/{siteId}/users/manage/user/{userId}` -> the UI of a particular user to manage

But I agree it's all a moot point for now!